Uploading release to Pypi fails

Trying to upload a release to pypi fails with:

```
  UploadError

  HTTP Error 400: 'https://github.com/tropxy' is an invalid value for Author-email. Error: Use a valid email address See https://packaging.python.org/specifications/core-metadata for more information.

  at ~/.local/lib/python3.10/site-packages/poetry/publishing/uploader.py:216 in _upload
      212│                     self._register(session, url)
      213│                 except HTTPError as e:
      214│                     raise UploadError(e)
      215│ 
    → 216│             raise UploadError(e)
      217│ 
      218│     def _do_upload(
      219│         self, session, url, dry_run=False
      220│     ):  # type: (requests.Session, str, Optional[bool]) -> None
 - Uploading ocpp-0.11.0-py3-none-any.whl 100%
Error: Process completed with exit code 1.
```

See here the failing job:
https://github.com/mobilityhouse/ocpp/runs/4353946005?check_suite_focus=true

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -3,13 +3,13 @@ name = "ocpp"
 version = "0.11.0"
 description = "Python package implementing the JSON version of the Open Charge Point Protocol (OCPP)."
 authors = [
-	"André Duarte <andre@switch-ev.com>",
+	"André Duarte <andre15x@gmail.com>",
 	"Auke Willem Oosterhoff <aukewillem.oosterhoff@mobilityhouse.com>",
 	"Greg Lutostanski <greg.luto@gmail.com>",
 	"Jonathan Herrmann <jonathan.herrmann@mobilityhouse.com>",
 	"Laysa Uchoa <laysa.uchoa@gmail.com>",
 	"Oz N Tiram <oz.tiram@mobilityhouse.com>",
-	"Patrick Roelke <patrick@switch-ev.com>",
+	"Patrick Roelke <proelke@gmail.com>",
 	"Roger <roger.duran@mobilityhouse.com>",
 	"dx <dx@mobilityhouse.com>",
 	"Jan Vincke <jan.vincke@mobilityhouse.com>",

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -3,13 +3,13 @@ name = "ocpp"
 version = "0.11.0"
 description = "Python package implementing the JSON version of the Open Charge Point Protocol (OCPP)."
 authors = [
-	"André Duarte <https://github.com/tropxy>",
+	"André Duarte <andre@switch-ev.com>",
 	"Auke Willem Oosterhoff <aukewillem.oosterhoff@mobilityhouse.com>",
-	"Greg Lutostanski <https://github.com/lutostag>",
+	"Greg Lutostanski <greg.luto@gmail.com>",
 	"Jonathan Herrmann <jonathan.herrmann@mobilityhouse.com>",
 	"Laysa Uchoa <laysa.uchoa@gmail.com>",
-	"Oz N Tiram <https://github.com/oz123>",
-	"Patrick Roelke <https://github.com/proelke>",
+	"Oz N Tiram <oz.tiram@mobilityhouse.com>",
+	"Patrick Roelke <patrick@switch-ev.com>",
 	"Roger <roger.duran@mobilityhouse.com>",
 	"dx <dx@mobilityhouse.com>",
 	"Jan Vincke <jan.vincke@mobilityhouse.com>",

